boot/kernel_os_test.go

```diff
@@ -457,7 +457,7 @@ func (s *ubootSuite) forceUbootBootloader(c *C) {
 	bootloader.Force(nil)
 
 	mockGadgetDir := c.MkDir()
-	err := ioutil.WriteFile(filepath.Join(mockGadgetDir, "uboot.conf"), nil, 0644)
+	err := ioutil.WriteFile(filepath.Join(mockGadgetDir, "uboot.conf"), []byte{1}, 0644)
```

Comment:
the whole suite probably needs a comment that while is testing the UC16/18 variant, the tested code about kernel extraction it the same also for UC20. or we need two suites?

Reply:
ah I think you're right we should really have 2 suites here sorry I overlooked this when fixing the one test we currently have